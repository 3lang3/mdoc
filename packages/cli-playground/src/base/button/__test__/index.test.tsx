import React from 'react';
import renderer from 'react-test-renderer';
import Button from '..';

it('Button default render', () => {
  const component = renderer.create(<Button>Facebook</Button>);
  let tree = component.toJSON() as any;
  expect(tree).toMatchSnapshot();
});
