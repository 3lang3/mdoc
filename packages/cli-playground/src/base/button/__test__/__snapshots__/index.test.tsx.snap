// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button default render 1`] = `
<div>
  Facebook
</div>
`;
